src/FreeRTOSConfig.h

```diff
@@ -60,7 +60,7 @@
 #define configUSE_TICKLESS_IDLE_SIMPLE_DEBUG    0 /* See into vPortSuppressTicksAndSleep source code for explanation */
 #define configCPU_CLOCK_HZ                      (SystemCoreClock)
 #define configTICK_RATE_HZ                      1024
-#define configMAX_PRIORITIES                    (3)
+#define configMAX_PRIORITIES                    (4)
```

Comment:
why the change to 4 priority levels? (or do I understand this line wrong?)

Reply:
Oops! I forgot to commit 1 file ([src/libs/mynewt-nimble/porting/npl/freertos/src/nimble_port_freertos.c](https://github.com/InfiniTimeOrg/InfiniTime/pull/1037/files#diff-c8bbccd055837f9c9c627160edeccc6dcd4f398ad38c433b6fa2c290bdb7fc51)), sorry for this!

So, we need 4 priorities  : 
 - 0 : Display
 - 1 : System & timer
 - 2 : BLE host
 - 3 : BLE LL
This way, we ensure that BLE tasks priorities are still higher than anything else, and that Display won't prevent system and timer tasks from executing from time to time.